.github/workflows/docker-configure-build-push.yaml

```diff
@@ -36,6 +36,7 @@ on:
         required: true
 jobs:
   configure-build-push:
+    timeout-minutes: 1440
```

Comment:
make smaller

Reply:
default is 6 hours so I'm just removing the line